docs/changelog.md

```diff
@@ -1,4 +1,10 @@
 # TBD
+### Features
+* Added Prysm CL (beacon, validator) Launcher
+
+### Changes
+* The `WaitForBeaconClientAvailability` method alos checks if the returned status is `READY`, it means the node is synced
```

Comment:
recommend this as "....also checks if the returned status is `READY`, which means the node is synced"